pages/LANGUAGE.md

```diff
@@ -0,0 +1,27 @@
+
+## How Gno Differs from Go
+
+![Gno and Go differences](static/img/gno-language/go-and-gno.jpg)
+
+The composable nature of Go/Gno allows for type-checked interactions between contracts, making Gno.land safer and more powerful, as well as operationally cheaper and faster. Smart contracts on Gno.land are light, simple, more focused, and easily interoperable—a network of interconnected contracts rather than siloed monoliths that limit interactions with other contracts.
+
+![Example of Gno code](static/img/gno-language/code-example.jpg)
+
+## Gno Inherits Go’s Built-in Security Features
+
+Go supports secure programming through exported/non-exported fields, enabling a “least-authority” design. It is easy to create objects and APIs that expose only what should be accessible to callers while hiding what should not be simply by the capitalization of letters, thus allowing a succinct representation of secure logic that can be called by multiple users.
+
+Another major advantage of Go is that the language comes with an ecosystem of great tooling, like the compiler and third-party tools that statically analyze code. Gno inherits these advantages from Go directly to create a smart contract programming language that is safe and helps developers to write secure code relying on the compiler, parser, and interpreter to give warning alerts for common mistakes.
+
+## Gno Is Essential for the Wider Adoption of Web3
+
+Other smart contract programming languages like Solidity are designed for one purpose only (writing smart contracts) and are bound by the limitations of the EVM. In addition, developers have to learn several languages if they want to understand the whole stack or work across different ecosystems. On Gno.land, every part of the stack is written in Gno so it’s easy to understand the entire system just by studying a relatively small code base.
+
+Using Gno, developers can rapidly accelerate application development and adopt a modular structure by reusing and reassembling existing modules without building from scratch. They can embed one structure inside another in an intuitive way while preserving localism, and the language specification is simple, successfully balancing practicality and minimalism.
+
+The Go language is so well designed that the Gno smart contract system will become the new gold standard for smart contract development and other blockchain applications. As a programming language that is universally adopted, secure, composable, and complete, Gno is essential for the broader adoption of web3 and its sustainable growth.
```

Comment:
This page needs to be completed, and I suggest removing it or replacing it with something shorter with a disclaimer.

Ideally, we should update this kind of docs on the main repo, and www.gno.land should link to the up-to-date documentation or synchronize.

Reply:
Let me ask to Christina, she is the one who wrote the text and I don't know what was her needs/goals here.

Reply:
synchronized is better for UX as we are currently redirecting our users in so many different websites, that looks similar  but without the same navbar. It becomes tricky to navigate in these situations.

Reply:
@moul It seems that Adriana requested it as she needs it for applying for the TM. Christina will reach you out about that.